doc/components/inventory.md

```diff
@@ -262,6 +271,163 @@ class SomeCmp extends React.Component {
 }
 ```
 
+### InventoryTable as tree
+Since inventory table is regular table you can pass additional data to it to be rendered as tree table with collapsible rows and some specific data in such row.
+```JSX
+import React from 'react';
+import * as reactRouterDom from 'react-router-dom';
+import * as reactCore from '@patternfly/react-core';
+import * as reactIcons from '@patternfly/react-icons';
+import { PaginationRow } from 'patternfly-react';
+import { registry as registryDecorator } from '@red-hat-insights/insights-frontend-components';
+
+@registryDecorator()
+class SomeCmp extends React.Component {
+    constructor(props, ctx) {
+        super(props, ctx);
+        this.state = {
+            InventoryCmp: () => <div>Loading...</div>,
+            items: [{
+                id: 'some-id',
+                children: [1], // please select some kind of more specific ID
+                active: false // to indicate that parent is not expanded
+            }, {
+                account: true, // since inventory table is checking if account is set
+                isOpen: false,
+                title: <div>Blaaa</div>, // What to show in expanded row
+            }]
+        }
+        this.onExpandClick = this.onExpandClick.bind(this);
+        this.fetchInventory();
+    }
+
+    async fetchInventory() {
+        const { inventoryConnector, mergeWithEntities } = await insights.loadInventory({
+            react: React,
+            reactRouterDom,
+            reactCore,
+            reactIcons,
+            pfReact: { PaginationRow }
+        });
+
+        this.getRegistry().register({
+            ...mergeWithEntities()
+        });
+
+        const { InventoryTable, updateEntities } = inventoryConnector();
+        this.updateEntities = updateEntities;
+
+        this.setState({
+            InventoryCmp: InventoryTable
+        })
+    }
+
+    onExpandClick(_cell, row, key) {
+        const { items } = this.state;
+        items.find(item => item.id === key).active = !row.active;
+        // Not ideal, but for the sake of example it's fine
+        row.children.forEach(child => {
+            items.find(item => item.id === child.id).isOpen = !row.active;
+        });
+        this.setState({
+            items
+        });
+        this.context.store.dispatch(this.updateEntities(items));
```

Comment:
`this.context.store` seems to be undefined?

Reply:
Context is automatically bound to this by calling constructor with second parameter and then passing it to parent using super https://github.com/facebook/react/blob/master/packages/react/src/ReactBaseClasses.js#L21

Reply:
I know, `this.context` is bound properly, but `this.context.store` is undefined, where is it initialized?